citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/DbTokenStoreTest.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.diging.citesphere.core.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import edu.asu.diging.citesphere.core.model.oauth.impl.DbAccessToken;
+import edu.asu.diging.citesphere.core.repository.oauth.DbAccessTokenRepository;
+
+public class DbTokenStoreTest {
+    @Mock
+    private DbAccessTokenRepository dbAccessTokenRepository;
+    
+    @InjectMocks
+    private DbTokenStore managerToTest = new DbTokenStore(dbAccessTokenRepository, null);
+    
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+    
+    @Test
+    public void test_findTokensByUserName() {
+        String username = "name";
+        List<DbAccessToken> dbAccessTokenList = new ArrayList<>();
+        Mockito.when(dbAccessTokenRepository.findByUsername(username)).thenReturn(dbAccessTokenList);
+        Assert.assertEquals(dbAccessTokenList, managerToTest.findTokensByUserName(username));
```

Comment:
so what's this test testing? that there are no results? the test should be named like that and there also needs to be a test for when there are results.